lib/constants.js

```diff
@@ -72,4 +72,7 @@ module.exports = {
     permittedCapitalizedBuckets: {
         METADATA: true,
     },
+    // Default expiration value of the S3 pre-signed URL duration
+    // 604800 seconds (seven days).
+    defaultExpiry: 7 * 24 * 60 * 60,
```

Comment:
May be a more descriptive name? `defaultPreSignedURLExpiry` instead of `defaultExpiry`!